orttraining/orttraining/test/graph/gradient_graph_builder_test.cc

```diff
@@ -798,6 +800,156 @@ class PipelineSplitter {
   }
 };
 
+// TODO: move to a proper location for pipeline training
+struct PipelineBatchInfo {
+  std::vector<std::pair<int64_t, int64_t>> events;
```

Comment:
>events [](start = 43, length = 6)

a comment for explaining the structure of this?

first number is the event_id a job waits for 
second number is the event_id a job notifies

Reply:
agree. This would make it easier for other reviewing this too.